File: Source/Admin/Web.Angular/src/app/domain/health-risk/queries/HealthRiskById.ts
import { QueryRequest } from "../../../services/QueryRequest";

export class HealthRiskById extends QueryRequest {
    constructor(healthRiskId: string) {
        super('healthRiskById', 'Read.HealthRiskFeatures.Queries.HealthRiskById', 
        {
        'healthRiskId': healthRiskId
        });
    }
}

File: Source/Admin/Read/HealthRiskFeatures/Queries/HealthRiskById.cs
using System;
using System.Linq;
using Dolittle.Queries;

namespace Read.HealthRiskFeatures.Queries
{
    public class HealthRiskById : IQueryFor<HealthRisk>
    {
        readonly IHealthRisks _collection;
        public Guid HealthRiskId { get; set; }

        public HealthRiskById(IHealthRisks collection)
        {
            _collection = collection;
        }

        public IQueryable<HealthRisk> Query => _collection.Query.Where(h => h.Id == HealthRiskId);
    }
}